azure-mgmt-dns/src/main/java/com/microsoft/azure/management/dns/DnsRecordSet.java

```diff
@@ -337,6 +378,20 @@
             WithAttach<ParentT> withMetadata(String key, String value);
         }
 
+        /**
+         * The stage of the record set definition allowing to enable ETag validation.
+         *
+         * @param <ParentT> the return type of {@link WithAttach#attach()}
```

Comment:
on these, I've been replacing our old wording, like this one - which is not C# friendly - with just a more generic: 
```
@param <ParentT> the stage of the parent definition to return to after attaching this definition
```